BAM/src/main/java/com/revature/bam/controller/CurriculumController.java

```diff
@@ -40,123 +38,128 @@
 
 	@Autowired
 	CurriculumService curriculumService;
-	
+
 	@Autowired
 	CurriculumSubtopicService curriculumSubtopicService;
-	
+
 	@Autowired
 	SubtopicService subtopicService;
-	
-	@Autowired 
+
+	@Autowired
 	BatchService batchService;
 
-	public CurriculumService get(){
+	public CurriculumService get() {
 		return curriculumService;
 	}
-	
+
 	/***
-	 * @author Nam Mai
-	 * Method is needed for injecting mocked services for unit test
+	 * @author Nam Mai Method is needed for injecting mocked services for unit test
```

Comment:
This should be on multiple lines

Reply:
revised that java doc to be on seperate lines